backend/src/models/User.test.ts

```diff
@@ -74,14 +74,14 @@ describe('User', () => {
 
             try {
                 await user.loadFromDatabase();
-                expect(true).toBeFalsy();
+                fail();
```

Comment:
All `fail()` calls need to be changed back to `expect(true).toBeFalsy()` as `fail()` is defined by Jest but not implemented after v27.0. This was also fixed as part of the previous refactoring PR.

Reply:
The previous changes can be found here: https://github.com/Anonymous-Voting-App/Anonymous-Voting-App/pull/78/commits/31f5c0e605187caa350367f4e5e3ac4e09c4fd13